cmd/deletion-reminder/Makefile

```diff
@@ -0,0 +1,14 @@
+DOCKER_TAG = docker.io/prombench/deletion-reminder:1.0.0
+
+build:
+	@go version | grep go1.11 || exit  "Requires golang 1.11 with support for modules!"
+	@GO111MODULE=on GOARCH=amd64 CGO_ENABLED=0 GOOS=linux go build -o deletion-reminder github.go reminder.go
```

Comment:
maybe try using `*.go` to avoid hard coding file names.

I even think it is fine to have just a single main.go file. it won't be that big.